types/lodash/lodash-tests.ts

```diff
@@ -5396,7 +5161,7 @@ fp.now(); // $ExpectType number
 
     _.chain("abc").get(1); // $ExpectType LoDashExplicitWrapper<string>
     _.chain("abc").get(["0"], "_");
-    _.chain([42]).get(0, -1); // $ExpectType LoDashExplicitWrapper<number | undefined>
+    _.chain([42]).get(0, -1); // ExpectType LoDashExplicitWrapper<number>
```

Comment:
This test was failing even on the default branch (probably due to a bug fix in TS2.9) so I had to disable the `$ExpectType`.